extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -67,7 +67,7 @@ export class SchemaCompareOptionsDialog {
 		this.schemaComparison.setDeploymentOptions(this.optionsModel.deploymentOptions);
 
 		if (this.optionsChanged) {
-			vscode.window.showWarningMessage(loc.OptionsChangedMessage, loc.YesButtonText, loc.NoButtonText).then((result) => {
+			vscode.window.showWarningMessage(loc.OptionsChangedMessage, { modal: true }, loc.YesButtonText).then((result) => {
```

Comment:
can we have Yes and No as options? Cancel is misleading here, I would think cancel means cancel the changes I made to the options.

Reply:
good point! I updated the options to yes and no 
![image](https://user-images.githubusercontent.com/31145923/134072729-03351fef-437b-459a-9b36-e991776a70ec.png)